### PR TITLE
Slack alert integration

### PR DESCRIPTION
**Pull request by upatricck**
## Done
* Added Sesh_Organisation and Slack_Channel models
* Functionality to report alerts on a specified alert channel

The slack authentication token for a sesh_organisation must be provided when creating the organisation

### CODE REVIEW

File: sesh/settings.py [hunk 1]
```diff
@@ -77,6 +78,9 @@
 # weather key
 FORECAST_KEY = config.get('api','forecast_key')
 
+# slack key
+SLACK_KEY = config.get('api', 'slack_key')
```
Comment (jlark) [inline sesh/settings.py]: I though this key was tied to oragnization?

File: seshdash/utils/send_slack.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            pass
+        else:
+            channel = '#' + channel
+     
+              
+        try:
+            # Posting the message
+            self.slack_client.chat.post_message(channel, message)
+            return True
+        except Exception, e:
```
Comment (jlark) [inline seshdash/utils/send_slack.py]: Here you need to actually check which exception is being thrown.
So if there is a specific slack exception that says channel does not exist you should catch that and do the necessary steps to create the channel. `except Exception,e` will create a channel on all exceptions regardless of the exception.

File: seshdash/utils/send_slack.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+              
+        try:
+            # Posting the message
+            self.slack_client.chat.post_message(channel, message)
+            return True
+        except Error, e: 
+            # Try creating the channel and post the message
+            self.slack_client.channels.create(channel)
+            self.slack_client.chat.post_message(channel, message)
+            logging.error('Slack Channel not Found')
```
Comment (jlark) [inline seshdash/utils/send_slack.py]: It's not really an error. More of a warning. Warning `slack channel not found creating ` would be betters suited

Comment (upatricck) [inline seshdash/utils/send_slack.py]: On it

**Pull request merged**